packages/date-picker/docs/DatePicker.stories.tsx

```diff
@@ -168,10 +161,3 @@ const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
 export const StickerSheetDefault = StickerSheetTemplate.bind({})
 StickerSheetDefault.storyName = "Sticker Sheet"
 StickerSheetDefault.parameters = { chromatic: { disable: false } }
-StickerSheetDefault.play = async ({ canvasElement }) => {
```

Comment:
act(..) error within production. Removed this story for now and have made a card to add back in using pseudo classes for storybook later.